assets/src/blocks/SecondaryNavigation/SecondaryNavigationEditor.js

```diff
@@ -0,0 +1,64 @@
+import {useEffect} from '@wordpress/element';
+import {getHeadingsFromBlocks} from '../TableOfContents/getHeadingsFromBlocks';
+import {initializeJustifyContentAdjustment} from './adjustNavWidth';
+
+const {useSelect} = wp.data;
+const {InspectorControls} = wp.blockEditor;
+const {PanelBody} = wp.components;
+const {__} = wp.i18n;
+
+const renderEdit = () =>  (
```

Comment:
Maybe we should just comment this out for now, until the documentation is updated? It's weird to redirect to another block's docs